src/main/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacade.java

```diff
@@ -876,16 +941,26 @@ public final Response getEventOverviews(@Context final HttpServletRequest reques
         }
 
         final Map<String, Constants.SortOrder> sortInstructions = Maps.newHashMap();
-        sortInstructions.put(EVENT_DATE_FIELDNAME, SortOrder.DESC);
+        sortInstructions.put(DATE_FIELDNAME, SortOrder.DESC);
 
         fieldsToMatch.put(TYPE_FIELDNAME, Arrays.asList(EVENT_TYPE));
 
         Map<String, AbstractFilterInstruction> filterInstructions = null;
-        if (null == showActiveOnly || showActiveOnly) {
+        if (showActiveOnly != null && showActiveOnly && showInactiveOnly != null && showInactiveOnly) {
+            return new SegueErrorResponse(Status.BAD_REQUEST,
+                    "You cannot request both show active and inactive only.").toResponse();
+        }
+        if (showActiveOnly != null && showActiveOnly) {
```

Comment:
This used to be an OR, meaning that if nothing was provided there was a default case. Does it work if neither `showActiveOnly` or `showInactiveOnly` are provided?